lib/option.rb

```diff
@@ -1,26 +1,38 @@
 # frozen_string_literal: true
 
 module Option
-  Provider = Struct.new(:name, :display_name) do
-    self::HETZNER = "hetzner"
-  end
-  Providers = [
-    [Provider::HETZNER, "Hetzner"]
-  ].map { |args| [args[0], Provider.new(*args)] }.to_h.freeze
+  provider_information = YAML.load_file("config/provider_information.yml")
```

Comment:
let's add `require "yaml"` to the start of the file. Otherwise, this fails in my local.